sport/app/football/model/DotcomRenderingFootballDataModel.scala

```diff
@@ -294,7 +294,7 @@ object DotcomRenderingFootballTablesDataModel {
             "entries" -> getEntries(table.competition, group),
           )
         },
-        "hasGroups" -> table.hasGroups,
+        "hasGroups" -> table.multiGroup,
```

Comment:
There are 2 types in frontend: hasGroups and multiGroup. hasGroups indicates that the request is for a specific group, and multigroup indicates that the the table has multiple groups or if hasGroups is true. 

Looking into the templates, they seem to always use multiGroup, so we don't need to pass them both to DCR.